corehq/apps/ota/views.py

```diff
@@ -84,7 +84,7 @@ def restore(request, domain, app_id=None):
         return HttpTooManyRequests()
 
     response, timing_context = get_restore_response(
-        domain, request.couch_user, app_id, **get_restore_params(request))
+        domain, request.couch_user, app_id, **get_restore_params(request, domain))
```

Comment:
If you're not sure if this is tested, one way to find out would be to put something obviously wrong on this line (like change `domain` to `blah` and then run the tests for this module.

Reply:
good point. trying this now

Reply:
confirming that this is tested!